sensor_msgs/PointCloud2 cloud_to_transform
---
sensor_msgs/PointCloud2 cloud_transformed
